src/core/namespace.c

```diff
@@ -1547,6 +1558,8 @@ static int mount_image(
                                        strempty(host_os_release_sysext_level),
                                        host_os_release_confext_level ? image_class_info[IMAGE_CONFEXT].level_env_print : "",
                                        strempty(host_os_release_confext_level));
+        if (r == -ENOCSI && m->required_class >= 0)
```

Comment:
ENOCSI seems exotic enough (never heard about that error before), hence I think the rhs of this check can go